custom-routes-vis/README.md

```diff
@@ -6,4 +6,4 @@ It utilizes the Maps API and Java Servlets as the backend.
 2. Deploy to custom-routes-visualization.appspot.com with
 `mvn clean package appengine:deploy`
 
-Set the environment variable DIRECTIONS_API_KEY to be the api key before running.
+Set the environment variable by running `export DIRECTIONS_API_KEY=apiKey` in the terminal before deploying project.
```

Comment:
```suggestion
Before running: Run `export DIRECTIONS_API_KEY=apiKey` in the terminal to set the environment variable.

Before deploying: Change line 50 of GetDirectionsServlet.java to set the actual Directions API key in the code, because the environment variable cannot be accessed in the deployed version. Change it back before pushing to git.
```